nixos/doc/manual/contributing-to-this-manual.chapter.md

```diff
@@ -11,3 +11,27 @@ $ nix-build nixos/release.nix -A manual.x86_64-linux
 ```
 
 If the build succeeds, the manual will be in `./result/share/doc/nixos/index.html`.
+
+**Contributing to the man pages**
+
+The manpages are written in [DocBook] which is XML.
+
+To see what your edits look like:
+
+```ShellSession
+$ cd /path/to/nixpkgs
+$ nix-build nixos/release.nix -A manpages.x86_64-linux
+```
+
+You can then read the man page you edited by running
+
+```ShellSession
+theCommand=nixos-rebuild  # Replace this with the command whose manual you edited
+man --manpath=result/share/man $theCommand
+```
```

Comment:
````suggestion
```
man --manpath=result/share/man nixos-rebuild # replace with the command whose man page you edited
```
````

Reply:
This doesn't make it clear *which* command to replace `man`, `nixos-rebuild`, or the entire line. I'll `sed s/this/nixos-rebuild` though. That should make it more clear

Reply:
I made it one line and amended the comment.

Could you have another look @ncfavier ?